main/src/com/google/refine/expr/functions/strings/Phonetic.java

```diff
@@ -48,12 +49,33 @@ SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
 
 public class Phonetic implements Function {
```

Comment:
This might be a good time to make sure that we've got at least cursory test coverage of the `phonetic` function. All we've got currently are a few invalid argument tests.

Reply:
done